tests/tests/test_monitor_client.py

```diff
@@ -470,6 +470,10 @@ def test_monitorclient_alert_email(self, monitor_commercial_setup_no_client):
             user_name,
             "OK: Monitor Alert for Log file contains " + log_pattern + " on " + devid,
         )
+        # in each CRITICAL and OK email we expect the pattern to be present at least 3 times
+        assert mail.count(log_pattern) >= 6
+        # in each CRITICAL and OK email we expect the log path to be present at least 1 time
+        assert mail.count(log_pattern) >= 2
```

Comment:
for a second I thought about parsing html, or comparing the decoded email body parts with a reference emails hardcoded in the test, but it would bring additional complexity as well as would bind us to the email format (dates in the footer, and possible layout changes, would make it a bit impractical). perhaps it is enough to ensure the number of times the path and pattern occur in the email.